README.md

```diff
@@ -25,28 +33,25 @@ pip install retworkx
 This will install a precompiled version of retworkx into your python
 environment.
 
-However, if there are no precompiled binaries published for your system
-you'll have to compile the code. The source package is also published on
-pypi so you can also run the above command to install it. However, there
-are 2 preconditions for this to work, first you need to have cargo/rustc
-**nightly** in your PATH. You can use [rustup](https://rustup.rs/) to
-make this step simpler. Secondly, you need to have `setuptools-rust`
-installed in your python environment. This can can be done by simply
-running:
+### Installing on a platform without precompiled binaries
 
-```bash
-pip install setuptools-rust
-```
-
-prior to running:
+If there are no precompiled binaries published for your system you'll have to
+build the package from source for your system. A source package is also
+published on pypi, so you still can also run the above `pip` command to install
+it. However, to be able able to build the package from the published source
```

Comment:
If you want to use however, then maybe put this sentence next to the one that you have to build from source (in which case, i'd rephrase it to "However, to do so..."

Reply:
Done in: https://github.com/Qiskit/retworkx/pull/91/commits/504ab0fc206d8e6c7c27474f89c223e00b139969